docs/OV_Runtime_UG/supported_plugins/CPU.md

```diff
@@ -1,139 +1,147 @@
 
-> **NOTE**: This pattern is possible only on CPUs with support of Streaming SIMD Extensions 4.2 
-> (SSE 4.2) and Intel AVX2 Instruction Set Architecture (ISA).
+> **NOTE**: When it comes to latency, one needs to keep in mind that running only one stream on multi-socket platform may introduce additional overheads on data transfer between NUMA nodes.
+> In that case it is better to run inference on one socket (please see [deployment optimization guide (additional configurations)](@ref openvino_docs_deployment_optimization_guide_dldt_optimization_guide_additional) for details).
 
-A combination of a group of a Convolution (or Binary Convolution) layer and simple layers and a group of a Depthwise Convolution
-layer and simple layers results in a single layer called *Convolution* (or *Binary Convolution*):
-> **NOTE**: Depthwise convolution layers should have the same values for the `group`, input channels, and output channels parameters.
+### Dynamic shapes
+CPU plugin provides full support for models with dynamic shapes. 
+But from the performance standpoint, it should be understood that the more degrees of freedom we have, the more difficult it is to achieve the best performance.
+The most flexible configuration is the fully undefined shape, when we do not apply any constraints to the shape dimensions. 
 
-![conv_depth_01]
+@snippet snippets/cpu/dynamic_shape.cpp undefined_shape
 
-### Fusing Convolution and Sum Layers
+In such configuration we will have high memory consumption since we can not estimate the total memory amount that can be allocated in advance and effectively reused.
+To reduce memory consumption through memory reuse, and as a result achieve better cache locality, which in its turn leads to better inference performance, it is better to use dynamic shapes with defined upper bounds.
 
-A combination of convolution, simple, and Eltwise layers with the sum operation results in a single layer called *Convolution*:  
+@snippet snippets/cpu/dynamic_shape.cpp defined_upper_bound
 
-![conv_sum_relu_01]
+> **NOTE**: Some runtime optimizations works better if the model shapes are known in advance. This means that for the best performance, it is better to use static shapes, of course if it is applicable to the specific problem.
 
-### Fusing a Group of Convolutions
+See [dynamic shapes guide](../ov_dynamic_shapes.md) for more details.
 
-If a topology contains the following pipeline, a CPU plugin merges split, convolution, and concatenation layers into a single convolution layer with the group parameter:   
+### Preprocessing acceleration
+CPU plugin supports a full set of the preprocessing operations, providing high performance implementations for them.
 
-![group_convolutions_01]
+See [preprocessing API guide](../preprocessing_overview.md) for more details.
 
-> **NOTE**: Parameters of the convolution layers must coincide.
+The CPU plugin precision conversion operation implementation supports the following element types:
```

Comment:
I guess it should be rephrased: "The CPU plugin support for Tensor precision conversion preprocessing is limited with the following ov::element types: "

Reply:
Done. Thanks.